deploy/karmada-dashboard-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/service-account-token
+metadata:
+  name: karmada-dashboard
+  namespace: karmada-system
+  annotations:
+    kubernetes.io/service-account.name: "karmada-dashboard"
```

Comment:
here need a new line, could you refine `deploy/karmada-dashboard-secret.yaml` file

Reply:
ok